#78 numero_primo/2019-1/enzo.rb

```diff
@@ -0,0 +1,9 @@
+module M
+  refine Integer do
+    def primo?
+      return true if self > 1 && (self % 1).zero? && (self % self).zero? && (2..self - 1).to_a.none? { |numero| (self % numero).zero? }
```

Comment:
@EnzoRobaina, you're the man! :1st_place_medal: :laughing:

Reply:
Tnxx 🚀